packages/connect-explorer/src/js/data/methods/management/wipeDevice.ts

```diff
@@ -0,0 +1,11 @@
+const name = 'wipeDevice';
+const docs = 'methods/wipeDevice.md';
```

Comment:
It looks like this variable `docs` is never used.

Reply:
fixed